src/peft/tuners/lora/eva.py

```diff
@@ -43,8 +43,14 @@ class _Hook:
     A base class for hooks that prepares layer inputs for EVA.
     """
 
-    def __init__(self, name: str, prepare_layer_inputs_fn: Optional[callable] = None):
+    def __init__(
+        self,
+        name: str,
+        prepare_layer_inputs_fn: Optional[callable] = None,
+        gather_distributed_inputs: bool = False,
```

Comment:
Since `gather_distributed_inputs` defaults to `True` now in `initialize_lora_eva_weights`, should it also be changed here?